src/service/Database.Api/Api/Api/DatabaseApi.cs

```diff
@@ -10,11 +10,23 @@ internal sealed partial class DatabaseApi : IDatabaseApi
 
     private const string SyncTimeFieldName = "SyncTime";
 
-    private readonly ISqlExecuteNonQuerySupplier sqlApi;
+    private const string DbAuditCreateTableQuery = """
+        IF NOT EXISTS (SELECT * FROM sysobjects WHERE name='_AuditDateTime' and xtype='U')
+            BEGIN
+            CREATE TABLE [_AuditDateTime]( 
+                [EntityName] varchar(100) NOT NULL,
+                [AuditDateTime] datetimeoffset NOT NULL,
+                PRIMARY KEY (EntityName)
```

Comment:
Let's add one more column:
[CreatedAt] datetimeoffset DEFAULT SYSDATETIMEOFFSET() NOT NULL